django/src/rdwatch_scoring/tasks/__init__.py

```diff
@@ -82,16 +92,32 @@ def get_siteobservations_images(
     # Ensure we are using ints for the DayRange and no_data_limit
     dayRange = int(dayRange)
     no_data_limit = int(no_data_limit)
-    site_observations = Observation.objects.filter(
-        site_uuid=site_eval_id
-    )  # need a full list for min/max times
-    site_obs_count = Observation.objects.filter(
-        site_uuid=site_eval_id, sensor=baseConstellation
-    ).count()
     found_timestamps = {}
     max_bbox = [float('inf'), float('inf'), float('-inf'), float('-inf')]
+
     # Use the base SiteEvaluation extents as the max size
-    baseSiteEval = Site.objects.get(pk=site_eval_id)
+    if proposal:
+        site_observations = AnnotationProposalObservation.objects.filter(
+            annotation_proposal_site_uuid=site_eval_id
+        )  # need a full list for min/max times
+        site_obs_count = AnnotationProposalObservation.objects.filter(
+            annotation_proposal_site_uuid=site_eval_id, sensor_name=baseConstellation
+        ).count()
+
+        site_db_model = AnnotationProposalSite
+        baseSiteEval = AnnotationProposalSite.objects.get(pk=site_eval_id)
+        geometry = baseSiteEval.geometry
+    else:
+        site_observations = Observation.objects.filter(
+            site_uuid=site_eval_id
+        )  # need a full list for min/max times
+        site_obs_count = Observation.objects.filter(
+            site_uuid=site_eval_id, sensor=baseConstellation
+        ).count()
+
+        site_db_model = Site
+        baseSiteEval = Site.objects.get(pk=site_eval_id)
+        geometry = baseSiteEval.union_geometry
```

Comment:
In the interest of keeping the `get_siteobservations_images` API the same between the `rdwatch` and `rdwatch_scoring` apps, would something like this work?

```suggestion
    try:
        site_db_model = AnnotationProposalSite
        baseSiteEval = AnnotationProposalSite.objects.get(pk=site_eval_id)
        geometry = baseSiteEval.geometry
        site_observations = AnnotationProposalObservation.objects.filter(
            annotation_proposal_site_uuid=site_eval_id
        )  # need a full list for min/max times
        site_obs_count = AnnotationProposalObservation.objects.filter(
            annotation_proposal_site_uuid=site_eval_id, sensor_name=baseConstellation
        ).count()
    except AnnotationProposalSite.DoesNotExist:
        site_db_model = Site
        baseSiteEval = Site.objects.get(pk=site_eval_id)
        geometry = baseSiteEval.union_geometry
        site_observations = Observation.objects.filter(
            site_uuid=site_eval_id
        )  # need a full list for min/max times
        site_obs_count = Observation.objects.filter(
            site_uuid=site_eval_id, sensor=baseConstellation
        ).count()
```

Any thoughts @JHUAPL-hjg1?

Reply:
@mvandenburgh What's the benefit of `keeping the get_siteobservations_images API the same`? 
`proposal` was added as a parameter to almost all endpoints and functions for `scoring`

Reply:
Pushed an update to address this: https://github.com/ResonantGeoData/RD-WATCH/pull/392/commits/5f73568fe561939fd1f5636214470353c6b5bd94